wasm/jit/jit_amd64.go

```diff
@@ -1643,6 +1865,267 @@ func (c *amd64Compiler) compileSqrt(o *wazeroir.OperationSqrt) error {
 	return nil
 }
 
+func (c *amd64Compiler) compileI32WrapFromI64() error {
+	target := c.locationStack.peek() // Note this is peek!
+	if err := c.ensureOnGeneralPurposeRegister(target); err != nil {
+		return err
+	}
+
+	mov := c.newProg()
+	mov.As = x86.AMOVL
+	mov.From.Type = obj.TYPE_REG
+	mov.From.Reg = target.register
+	mov.To.Type = obj.TYPE_REG
+	mov.To.Reg = target.register
+	c.addInstruction(mov)
+	return nil
+}
+
+func (c *amd64Compiler) compileITruncFromF(o *wazeroir.OperationITruncFromF) error {
+	return nil
```

Comment:
I will complete ITruncFromF in another PR as it should be fairly complex and would double the size of this PR :D